backup/restore.go

```diff
@@ -301,5 +301,13 @@ func (rm *RestoreManager) applyBinlog(ctx context.Context, op bkop.Operator, key
 		return fmt.Errorf("zstd exited abnormally: %w", err)
 	}
 
-	return op.LoadBinlog(ctx, binlogDir, rm.restorePoint)
+	mysqlBinlogTmpdir := filepath.Join(rm.workDir, "tmp")
```

Comment:
Compared to other variables in this function, this name is descriptive.
A simple name would be better here.

```suggestion
	tmpDir:= filepath.Join(rm.workDir, "tmp")
```